wpilibc/src/main/native/cpp/smartdashboard/SendableBuilderImpl.cpp

```diff
@@ -141,76 +141,139 @@ void SendableBuilderImpl::AddPropertyImpl(Topic topic, Getter getter,
   m_properties.emplace_back(std::move(prop));
 }
 
+template <typename Topic, typename Value>
+void SendableBuilderImpl::PublishConstImpl(Topic topic, Value value) {
+  auto prop = std::make_unique<PropertyImpl<Topic>>();
+  prop->pub = topic.Publish();
+  prop->updateNetwork = [value, prevPub = NT_Publisher{}](
+                            auto& pub, int64_t time) mutable {
+    if (prevPub != pub.GetHandle()) {
+      pub.Set(value, time);
```

Comment:
I'm confused as to how this is supposed to work.  It's capturing value by value, so it's just going to output the same value over and over?

Reply:
The value is constant, and will be sent only once per publisher.